test/browser/clubs-page.test.jsx

```diff
@@ -97,6 +97,7 @@ describe("ClubsPage.ClubList", function() {
     owner: 'foo',
     website: 'http://example.org',
     location: 'Somewhere, USA',
+    status: 'approved',
     name: 'foo club'
   }];
 
```

Comment:
Good idea--I'd like to wait until we have the OK from sabrina on the UX, but I will definitely add that!